packages/core/src/events/EventHandlers.ts

```diff
@@ -1,9 +1,21 @@
-import { NodeId, Node, Indicator } from "../interfaces";
+import { createShadow } from "./createShadow";
+import { NodeId, Indicator, Tree } from "../interfaces";
 import { Handlers, ConnectorsForHandlers } from "@craftjs/utils";
 import { debounce } from "debounce";
 import { EditorStore } from "../editor/store";
 
-type DraggedElement = NodeId | Node;
+type DraggedElement = NodeId | Tree;
+
+const event = ({
```

Comment:
Oh I like that this method. But could we have it follow a similar pattern like `addEventListener` instead of having to specify the object properties name/handler/capture? 

So like:
```jsx
const addEventListener = (name, handler, capture)  => {...}
```

Also, could you move this to `packages/utils/src/Handlers.ts` so that we can use this in the layers package as well.